src/sanitation.js

```diff
@@ -184,10 +184,10 @@ export const WHITELISTED_ATTRS = [
   'subscriptions-display',
   'subscriptions-section',
   'subscriptions-service',
-  // Attributes for amp-drilldown.
-  'amp-drilldown-submenu',
-  'amp-drilldown-submenu-open',
-  'amp-drilldown-submenu-close',
+  // Attributes for amp-nested-menu.
```

Comment:
@choumx for owners bot? 
This is a strict renaming of an unlaunched component.

Reply:
LGTM